### PR TITLE
Modularize server

### PR DESCRIPTION
**Pull request by Chadtech**

### CODE REVIEW

File: server/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
-name = "server-actix"
+name = "server"
 version = "0.1.0"
-authors = ["loopy"]
```
Comment (Chadtech) [inline server/Cargo.toml:4]: @VictorXLR were you signing you name "loopy"?

File: server/src/main.rs [hunk 2]
```diff
@@ -1,51 +1,30 @@
...
 }
 
-// GET /again
+/// GET /again
 async fn index2() -> impl Responder {
     HttpResponse::Ok().body("Hello World Again")
 }
 
-// GET /games/count, we also pass in the state
+/// GET /games/count, we also pass in the state
```
Comment (Chadtech) [inline server/src/main.rs:22]: I think triple forward slash is a Rust standard for documentation. It gets highlighted differently in my IDE too. Just FYI @VictorXLR

File: server/src/domain/mod.rs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+pub mod game;
+pub mod model;
+pub mod player;
```
Comment (Chadtech) [inline server/src/domain/mod.rs:3]: I guess in Rust to make a module you need to declare it as a `mod`. This `domain/mod.rs` file declares all the things in `domain` as modules.

File: server/src/main.rs (hunk 2 above)
Comment (VictorXLR) [reply server/src/main.rs:22]: oh, i didnt know that :), ill Change it moving forward

File: server/Cargo.toml (hunk 1 above)
Comment (VictorXLR) [reply server/Cargo.toml:4]: No, My system user name is `loopy`, Ill change that to VictorXLR now

**Pull request merged**